docs/manuals/gui/viselements/charts/funnel.md

```diff
@@ -0,0 +1,359 @@
+# Funnel charts
+
+Funnel charts represent a series of decreasing values as shrinking stacked bars.
+
+Funnel charts are typically used:
+
+- To visualize the changed value at each stage of a process.
+- To estimate a potential figure at every stage.
+- To track conversion and retention rates.
+- To reveal bottlenecks in a linear process.
+- To track shopping cart workflows.
+- To track the progress and success of click-through advertising and marketing campaigns.
+
+To create a funnel chart in Taipy, you must set the [*type*](../chart.md#p-type) property
+of the chart control "funnel" or "funnelarea".
+
+## Key properties
+
+| Name            | Value           | Notes   |
+| --------------- | ------------------------- | ------------------ |
+| [*type*](../chart.md#p-type)      | `funnel` or `funnelarea`  |  |
+| [*x*](../chart.md#p-x)            | Volume values for stages  | For the `funnel` type.  |
+| [*y*](../chart.md#p-y)            | Stage names  | For the `funnel` type.  |
+| [*values*](../chart.md#p-values)  | Volume values for stages  | For the `funnelarea` type.  |
+| [*text*](../chart.md#p-text)      | Stage names  | For the `funnelarea` type.  |
+| [*marker*](../chart.md#p-marker)  | dictionary  | `color` can be used to indicate the color of the box representing a stage.<br/>`line` provides the parameters for styling the box frames. |
+| [*options*](../chart.md#p-options)  | dictionary  | `connector` can be used to customize the graphical properties of the lines connecting the boxes. |
+
+## Examples
+
+### Simple funnel chart {data-source="gui:doc/examples/charts/funnel-simple"}
```

Comment:
data-source has no extension ?

Reply:
No need.
For the time being, it will be a raw .py file. In the future, a directory?